examples/scroll_compressor_multilump.py

```diff
@@ -0,0 +1,265 @@
+#
+#     FP = FlowPath(key1='outlet.1',
+#                   key2='dd',
+#                   MdotFcn=IsentropicNozzleWrapper(),
+#                   )
+#     FP.A = pi*0.006**2/4
+#     ScrollComp.add_flow(FP)
+#
+#     FP = FlowPath(key1='outlet.1',
+#                   key2='ddd',
+#                   MdotFcn=IsentropicNozzleWrapper(),
+#                   )
+#     FP.A = pi*0.006**2/4
+#     ScrollComp.add_flow(FP)
+
+    ScrollComp.add_flow(FlowPath(key1='d1',
+                                 key2='dd',
+                                 MdotFcn=ScrollComp.D_to_DD))
+    ScrollComp.add_flow(FlowPath(key1='d2',
+                                 key2='dd',
+                                 MdotFcn=ScrollComp.D_to_DD))
+
+    #Connect the callbacks for the step, endcycle, heat transfer and lump energy balance
+    ScrollComp.connect_callbacks(step_callback = ScrollComp.step_callback,
+                                 endcycle_callback = ScrollComp.endcycle_callback,
+                                 heat_transfer_callback = ScrollComp.heat_transfer_callback,
+                                 lumps_energy_balance_callback = ScrollComp.lump_energy_balance_callback
+                                 )
+
+    from time import clock
+    t1=clock()
+    ScrollComp.RK45_eps = 1e-6
+    ScrollComp.eps_cycle = 3e-3
+    ScrollComp.verbosity = 10
+
+    ScrollComp.precond_solve(key_inlet='inlet.1',
+                                key_outlet='outlet.2',
+                                solver_method='RK45',
+                                OneCycle = OneCycle,
+                                plot_every_cycle= False,
+                                x0 = [330,330,350], #Guesses [Td,Tlump[0],Tlump[1]]
+                                #hmin = 1e-3
+                                eps_cycle = 3e-3,
+                                eps_energy_balance = 0.1 #relaxed multi-lump convergence
+                                )
+
+    print('time taken', clock()-t1)
+
+    debug_plots(ScrollComp)
+
+    #Calculate cooling capacity
+    Tcond_out = Tcond - DT_sc #K
+    hcond_out = CP.PropsSI('H','T',Tcond_out,'P',pc*1000,Ref) #J/kg
+    hevap_in = hcond_out/1000 #kJ/kg
+    hevap_out = inletState.h
+    Qdot_evap = ScrollComp.mdot*(hevap_out - hevap_in) #kW
+    print('Cooling capacity:',Qdot_evap,'kW')
+    print('Cooling capacity:',Qdot_evap*1000/0.2928104,'Btu/hr')
+
+    del ScrollComp.FlowStorage
+    from PDSim.misc.hdf5 import HDF5Writer
+    h5 = HDF5Writer()
+    h5.write_to_file(ScrollComp, HDF5file)
+
+    return ScrollComp
+
+def Single_Run(Te_F,Tc_F,Ref = 'R134a'):
+
+    Te_C = F2C(Te_F)
+    Tc_C = F2C(Tc_F)
+
+    filename = 'CompressorName_'+Ref+'_'+str(Te_F)+'_'+str(Tc_F)+'.h5'
+
+    Compressor(Te = Te_C,
+                DTsh = 11.1,
+                Tc = Tc_C,
+                Tamb = 25,
+                Nmot = 3600,
+                f = None,
+                OneCycle = False,
+                Ref = Ref,
+                HDF5file=filename)
```

Comment:
Please remove, or at the very least, no Fahrenheit :)

Reply:
I had it setup of a model validation with a full map and I got lazy to modify it. I will update the example